.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release
+on:
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
I'm not familiar with GitHub Actions, but does this work?

According to the following discussion, it does only works for Workflow templates:
https://stackoverflow.com/questions/64781462/github-actions-default-branch-variable

Reply:
`$default-branch` would be `dev` then?

Reply:
Yes

Reply:
Oh, I got confused a bit. So indeed `dev` is a default branch. However, it is not the branch which has the latest release. The `master` branch is the one. In that case Release PR should go from `dev` into `master`. That was my workflow so far.

Reply:
But what is the current workflow? Maybe it would be good to migrate repo to the current elementary workflow.

Reply:
The workflow we do is the development head is `main` and releases go into a branch named after the current distro channel (so for this release it would be `noble`). That gets changed when the app becomes no longer compatible with the oldest supported distro channel.

So for example we have a `jammy` branch of an app and then we depend on newer libraries so we start releasing into `noble`. But if we want to backport a fix from `main` to `jammy` that's possible even though our current release branch is now `noble`. Hope that makes sense!

Reply:
But how  the Release PR is prepared? From what branch changes are taken, since everything is already merged into `main`?